lib/src/components/NxTextInput/stateful/types.ts

```diff
@@ -6,10 +6,10 @@
  */
 import * as PropTypes from 'prop-types';
 
-import { NxTextInputType, HTMLProps, propTypes as nxTextInputPropTypes, Validator } from '../types';
+import { PublicNxTextInputType, HTMLProps, propTypes as nxTextInputPropTypes, Validator } from '../types';
 
 export type Props = HTMLProps & {
-  type?: NxTextInputType | null;
+  type?: PublicNxTextInputType | null;
```

Comment:
This shouldn't work with the way you've got `NxStatefulDateInput` implemented :thinking:

Reply:
> This shouldn't work with the way you've got `NxStatefulDateInput` implemented 🤔

Fixed: c6e1f4b31240cf0571402551b86aff7470a0e989